File: nordeus_challenge_backend_spring_boot/src/main/java/com/mmileticc/nordeuschallenge/model/dto/MoveDTO.java
package com.mmileticc.nordeuschallenge.model.dto;

import com.mmileticc.nordeuschallenge.model.entity.enums.BuffAttribute;
import com.mmileticc.nordeuschallenge.model.entity.enums.BuffTarget;
import com.mmileticc.nordeuschallenge.model.entity.enums.EffectType;
import com.mmileticc.nordeuschallenge.model.entity.enums.MoveType;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MoveDTO {

    private Long id;
    private String name;
    private MoveType moveType;
    private EffectType effectType;
    private int power;
    private BuffAttribute buffAttribute;
    private BuffTarget buffTarget;
    private int buffAmount;
    private int buffDuration;
}

File: Gauntlet of the Hero - GOTH/Assets/_Scripts/Models/Backend/MoveDto.cs
using Newtonsoft.Json;

namespace GauntletOfTheHero.Backend.Models
{
    [System.Serializable]
    public class MoveDto
    {
        [JsonProperty("id")]
        public long id;

        [JsonProperty("name")]
        public string name;

        [JsonProperty("moveType")]
        public MoveType moveType;

        [JsonProperty("effectType")]
        public EffectType effectType;

        [JsonProperty("power")]
        public int power;

        [JsonProperty("buffAttribute")]
        public BuffAttribute? buffAttribute;

        [JsonProperty("buffTarget")]
        public BuffTarget? buffTarget;

        [JsonProperty("buffAmount")]
        public int buffAmount;

        [JsonProperty("buffDuration")]
        public int buffDuration;
    }
}